### ISSUE TITLE
Revert back to setting mtime to 0, not 1, to follow what OSTree does 

### ISSUE THREAD

**Issue by mariospr** [flatpak/flatpak#291]
Flatpak sets mtime to 1 following a change introduced in OSTree 2016.6, but that's not longer the case as it got reverted today: https://github.com/ostreedev/ostree/commit/845dc651962b2c106872428336c9d81c2ed12432

In Endless, having a different criteria on OSTree and flatpak is troublesome, since the building machinery that creates the OSTree for the OS's root filesystem is also used to create our own com.endlessm.Platform/Sdk runtimes, reusing many steps and customization hooks.

For instance, if OSTree sets mtime to 0 and we create our base rootfs for the runtime with that (where we pre-generate caches like the one for Fontconfig) and then flatpak deploys it setting mtime to 1, that would screw the whole cache handling, forcing flatpak applications to ignore the pregenerated caches included with the runtime, thus suffering of a noticeable delay when starting up for the first time.

So, in summary, it would be great if flatpak could follow suit on this revert too, to keep things consistent.

**Comment by alexlarsson** [comment]
Yes ,this is reverted in master
